examples/02_meta-analyses/10_plot_cbmr.py

```diff
@@ -0,0 +1,362 @@
+"""
+
+.. _metas_cbmr:
+
+===========================================
+Coordinate-based meta-regression algorithms
+===========================================
+
+A tour of CBMR algorithms in NiMARE
+
+This tutorial is intended to provide a brief description and example of the CBMR
+algorithm implemented in NiMARE. For a more detailed introduction to the elements
+of a coordinate-based meta-regression, see other stuff.
```

Comment:
I think before we jump in we need to give more context as to what we're doing, and why.

I would give a brief high-level introduction to CBMR, and a more detailed overview as to what we're going to see in this example notebook.

Reply:
Thanks for the suggestions! I've added both high-level and detailed introduction to CBMR in the example file.